boards/sparc/gr716a_mini/doc/index.rst

```diff
@@ -0,0 +1,216 @@
+.. _gr716a_mini:
+
+GR716-MINI Development Board
+############################
+
+Overview
+********
+
+The GR716-MINI development board provides:
+
+* GR716 microcontroller
+* SPI Flash PROM, 32 MiB
+* SRAM, 2 MiB
+* FTDI USB interface for UART debug link (AHBUART) and application UART
+  (APBUART).
+* 4x MMCX connectors (2 ADC, 2 DAC)
+* Miniature 80 pin mezzanine connector (bottom side)
+
+.. figure:: gr716a_mini.jpg
+   :width: 543px
+   :align: center
+   :alt: GR716-MINI Development Board
+
+   GR716-MINI Development Board (Credit: Cobham Gaisler AB)
+
+Hardware
+********
+
+Console Output
+==============
+
+By default, the kernel is configured to send console output to the
+first APBUART peripheral (apbuart0). The UART debug forwarding setting,
+if enabled in GRMON, is preserved.
+
+Programming and Debugging
+*************************
+
+Building
+========
+
+Applications for the ``gr716a_mini`` board configuration can be built
+as usual (see :ref:`build_an_application`).
+In order to build the application for ``gr716a_mini``, set the ``BOARD``
+variable to ``gr716a_mini``.
+
+The applicatoin is linked to the on-chip tightly coupled memory by
```

Comment:
```suggestion
The application is linked to the on-chip tightly coupled memory by
```

Reply:
Thank you. I will correct the typo as suggested.